Docker/README.md

```diff
@@ -160,11 +160,12 @@ with HTTPS, for instance using [Let’s Encrypt](https://letsencrypt.org/).
 
 A [docker-compose.yml](docker-compose.yml) file is given as an example, using PostgreSQL. In order to use it, you have to adapt:
 - In the `postgresql` service:
-	* the `volumes` section;
+	* the `volumes` section. Be careful to keep the path `/var/lig/postgresql/data` for the container. If the path is wrong, you will not get any error but your db will be gone at the next run;
```

Comment:
I guess it is `/var/lib/postgresql/data` with a `b`, isn't it?

Reply:
Yep, sorry. It's corrected.